modules/microshift-low-latency-prepare-workload.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-prepare-workload_{context}"]
+= Preparing a {microshift-short} workload for low latency
+
+To take advantage of low latency, workloads running on {microshift-short} must have the `microshift-low-latency` container runtime configuration set using the `RuntimeClass` feature. The CRI-O `RuntimeClass` object is installed with the `microshift-low-latency` RPM, so only the pod annotations need to be configured.
+
+.Prerequisites
+* You installed the `microshift-low-latency` RPM package.
+//Does microshift need to be stopped?
+//anything else?
+
+.Procedure
+
+. Use the following example to set the following annotations in the pod spec:
++
+[source,text]
+----
+cpu-load-balancing.crio.io: "disable"
+irq-load-balancing.crio.io: "disable"
+cpu-quota.crio.io: "disable"
+----
++
+.Example pod that runs `oslat` test:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: oslat
+  annotations:
+    cpu-load-balancing.crio.io: "disable" <1>
+    irq-load-balancing.crio.io: "disable" <2>
+    cpu-quota.crio.io: "disable" <3>
+spec:
+  runtimeClassName: microshift-low-latency <4>
+  containers:
+  - name: oslat
+    image: quay.io/container-perf-tools/oslat
+    imagePullPolicy: Always
+    resources:
+      requests:
+        memory: "400Mi"
+        cpu: "10"
+      limits:
+        memory: "400Mi"
+        cpu: "10"
+    env:
+    - name: tool
+      value: "oslat"
+    - name: manual
+      value: "n"
+    - name: PRIO
+      value: "1"
+    - name: delay
+      value: "0"
+    - name: RUNTIME_SECONDS
+      value: "60"
+    - name: TRACE_THRESHOLD
+      value: ""
+    - name: EXTRA_ARGS
+      value: ""
+    securityContext:
+      privileged: true
+      capabilities:
+        add:
+          - SYS_NICE
+          - IPC_LOCK
+
+----
+<1> Disables the CPU load balancing for the pod.
+<2> Opts the pod out of interrupt handling (IRQ).
+<3> Disables the CPU completely fair scheduler (CFS) quota at the pod run time.
+<4> The `runtimeClassName` must match the name of the performance profile configured in the cluster.
++
+[IMPORTANT]
+====
+For the pod to have the `Guaranteed` QoS class, it must have the same values of CPU and memory in requests and limits. See link:https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#guaranteed[Guaranteed] (Kubernetes upstream documentation)
+====
+
+. To apply changes, restart the workload pods.
+
+//Do we need to restart microshift and cri-o, too?
```

Comment:
No, we don't need to restart workloads as well. If we create the manifest, they will be created. Not sure if we could update the manifest to have these settings required to have annotations and Guaranteed QoS - some fields cannot be changed in already running Pod